nft-standard.md

```diff
@@ -0,0 +1,64 @@
+## Preamble
+* Sip Number: 009
+* Title: Standard Trait Definition for Non-Fungibe Tokens
+* Author: Friedger Müffke (mail@friedger.de)
+* Consideration: Technical
+* Type: Standard
+* Status: Draft
+* Created: 2020-12-10
+* License: CC0-1.0
+* Sign-off:
+
+## Abstract
+Non-fungible token are unique digital assets that are registered on the Stacks blockchain through a smart contract with certain properties. 
+Users should be able to identify a single non-fungible token. Users should be able to own it and transfer it. Non-fungible tokens can have more properties 
+that are not specified in this standard.
+
+## License and Copyright
+
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at https://creativecommons.org/publicdomain/zero/1.0/
+This SIP’s copyright is held by the Stacks Open Internet Foundation.
+
+## Introduction
+
+
+## Specification
+
+```
+
+(define-trait stacks-token-nft-standard-v1
+  (
+    ;; Name, limited to 32 chars
+    (name () (string-ascii 32)) 
+
+    ;; Icon URL, limited to 2048 chars
+    (icon-url () (string-ascii 2048))
```

Comment:
Should we change this to 400,000 to support base64 encoded images for a size up to 400kb

Reply:
I'd say no - too much state, and adds complexity for clients. If anything, it might not even be desirable for many clients to show any arbitrary image, given the potential for abuse.